diff --git a/include/configs/evb_rk3399.h b/include/configs/evb_rk3399.h
index 70ab5877..0eff0a5d 100644
--- a/include/configs/evb_rk3399.h
+++ b/include/configs/evb_rk3399.h
@@ -7,9 +7,17 @@
 #define __EVB_RK3399_H
 
 #define ROCKCHIP_DEVICE_SETTINGS \
-		"stdin=serial,usbkbd\0" \
-		"stdout=serial,vidconsole\0" \
-		"stderr=serial,vidconsole\0"
+		"stdin=serial\0" \
+		"stdout=serial\0" \
+		"stderr=serial\0"
+
+#define CFG_EXTRA_ENV_SETTINGS \
+		"bootcmd=bootflow scan -lb\0" \
+		"kernel_addr_r=0x02000000\0" \
+		"fdt_addr_r=0x12000000\0" \
+		"ramdisk_addr_r=0x12180000\0" \
+		"boot_targets=mmc0\0" \
+		ROCKCHIP_DEVICE_SETTINGS
 
 #include <configs/rk3399_common.h>
 
diff --git a/include/configs/rk3399_common.h b/include/configs/rk3399_common.h
index 76f40e7c..ba8cb9b2 100644
--- a/include/configs/rk3399_common.h
+++ b/include/configs/rk3399_common.h
@@ -31,12 +31,14 @@
 	"ramdisk_addr_r=0x12180000\0"	\
 	"kernel_comp_size=0x8000000\0"
 
+#ifndef CFG_EXTRA_ENV_SETTINGS
 #define CFG_EXTRA_ENV_SETTINGS		\
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0"	\
 	"partitions=" PARTS_DEFAULT	\
 	ENV_MEM_LAYOUT_SETTINGS		\
 	ROCKCHIP_DEVICE_SETTINGS	\
 	"boot_targets=" BOOT_TARGETS "\0"
+#endif
 
 #endif /* CONFIG_XPL_BUILD */
 
